### ISSUE TITLE
Governance structure & Code of Conduct

### ISSUE THREAD

**Issue by vollmersj** [alan-turing-institute/MLJ.jl#181]
We should move to 
- a governance structure
[https://github.com/cortexproject/cortex/blob/master/GOVERNANCE.md](https://github.com/cortexproject/cortex/blob/master/GOVERNANCE.md)
- code of conduct
[https://www.contributor-covenant.org/](https://www.contributor-covenant.org/)

**Comment by OkonSamuel** [comment]
@vollmersj Whats the status??

**Comment by ablaom** [comment]
Thanks @OkonSamuel  for reflagging this.

I think in a discussion some time ago with @aviks and IQVIA, with a number of MLJ stakeholders present, it was decided that:

- governance structure is overkill at this point

- code of conduct is a good idea

I don't think there was a strong opinion about what model to use, but I'd be happy to go with the Julia one: https://julialang.org/community/standards/

If I hear no objections by August 20th, 2020, I will take that as tacit approval adoption of the Julia code of conduct by all interested stake holders. It will then just be a matter of adding a link to the Julia code of conduct from our repo.

**Comment by vollmersj** [comment]
Yes I remember this discussion. I think we might want to adopt the numfocus code of conduct that is also adopted by SciML [see](https://sciml.ai/governance/).  Regarding governance - some concern was raised discourse regarding this. We have grown since these discussion - I think we should look into to include representatives from julia ecosystem in some form of 3 monthly meetings.

**Comment by ablaom** [comment]
> We have grown since these discussion 

Our paltry resources have not substantially changed. Monthly meetings with interested parties is fine with me. However, I am concerned that a formal governance structure will lead to the surrender of control to stakeholders without a corresponding commitment of resources. (Translation: "Show me the money!")
